cmd/mtaBuild_test.go

```diff
@@ -250,13 +296,15 @@ func TestMarBuild(t *testing.T) {
 			err := runMtaBuild(options, &cpe, utilsMock)
 
 			assert.Nil(t, err)
-			assert.Contains(t, utilsMock.Env, "MAVEN_OPTS=-Dmaven.repo.local=/root_folder/workspace/.pipeline/local_repo")
+			assert.Contains(t, utilsMock.Env, filepath.Join("MAVEN_OPTS=-Dmaven.repo.local=", "root_folder", "workspace", ".pipeline", "local_repo"))
```

Comment:
It looks a bit suspicious what `MAVEN_OPTS=-Dmaven.repo.local=` is also part of the join to get the leading `/`..

Reply:
guess it's fine as this is a test case